rs/bitcoin/adapter/src/get_successors_handler.rs

```diff
@@ -64,7 +63,10 @@ const_assert_eq!(
 );
 
 // Max height for sending multiple blocks when connecting the Bitcoin mainnet.
-const MAINNET_MAX_MULTI_BLOCK_ANCHOR_HEIGHT: BlockHeight = 750_000;
+pub(crate) const BTC_MAINNET_MAX_MULTI_BLOCK_ANCHOR_HEIGHT: BlockHeight = 750_000;
+
+// Max height for sending multiple blocks when connecting the Dogecoin mainnet.
+pub(crate) const DOGE_MAINNET_MAX_MULTI_BLOCK_ANCHOR_HEIGHT: BlockHeight = 5_000_000;
```

Comment:
Considering that Dogecoin mainnet height > 5_860_000, it should be safe enough to set this to 5_000_000.